metrics/cli/test/cli/deployer/deployer_test.dart

```diff
@@ -2119,19 +2189,18 @@ void main() {
         whenDirectoryExist().thenReturn(true);
         whenPromptToSetupSentry().thenReturn(true);
         when(firebaseService.configureAuthProviders(any)).thenReturn(clientId);
-        whenCreateSentryRelease()
-            .thenAnswer((_) => Future.value(sentryRelease));
+        whenGetSentryRelease().thenReturn(sentryRelease);
         when(sentryService.getProjectDsn(any)).thenReturn(sentryDsn);
         when(fileHelper.getFile(any)).thenReturn(file);
 
-        final sentryConfig = SentryConfig(
+        final sentryConfig = SentryWebConfig(
```

Comment:
```suggestion
        final sentryWebConfig = SentryWebConfig(
```
🔖  What do you think?